apps/web/components/examples/blockchain/api-client-demo.tsx

```
@@ -0,0 +1,368 @@
+  }
+
+  const testIndividualEndpoints = async () => {
+    const results = []
+
+    try {
+      // Test progress tracker
+      const progressResult = await client.progressTracker.markLessonComplete(userId, 1, 1)
+      results.push({ service: 'Progress Tracker', success: progressResult.success, error: progressResult.error })
+
+      // Test badge tracker
+      const badgeResult = await client.badgeTracker.mintBadge(userId, 'ChapterCompletion' as any, 1, 'Test badge')
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)